.storybook/decorators.tsx

```diff
@@ -58,10 +59,14 @@ export const withFormik: Decorator = (Story, context) => {
   );
 };
 
-export const withRenderSettingsProvider: Decorator = (Story, {parameters}) => (
-  <RendererSettingsProvider
+export const withRenderSettingsProvider: Decorator = (
+  Story,
+  {parameters, args: {components, componentDefinition}}
+) => (
+  <FormSettingsProvider
     requiredFieldsWithAsterisk={parameters?.renderSettings?.requiredFieldsWithAsterisk ?? true}
+    components={(components ?? componentDefinition ?? []) as AnyComponentSchema[]}
```

Comment:
this one looks a bit odd - `componentDefinition` sounds like a single component definition instead of a list of components

However, I've updated this to grab it from the `parameters` and fall back to an empty list - of course you can make changes to this decorator in your branch as needed, but for the baseline I'd like to keep it more strict initially.